custom/icds/const.py

```diff
@@ -32,3 +32,5 @@
 FILE_TYPE_CHOICE_DOC = 2
 DISPLAY_CHOICE_LIST = 1
 DISPLAY_CHOICE_FOOTER = 2
+
+IS_ICDS_ENV = settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS
```

Comment:
how about making this a setting so that we don't have imports from `custom` everywhere?

Reply:
that would be nice as well, will take in a follow up PR

Reply:
Actually thought about it and weighing in adv/disadv of the approaches reverted this change.
the idea of this constant was to avoid re-checking for settings but i see that having `custom` import isn't great either. Moving to a setting is the same as it's now. So just made it back as it was.